bot.py

```diff
@@ -0,0 +1,101 @@
+import discord
+from discord.ext import commands
+from discord.ui import Button, View
+import json
+from dotenv import load_dotenv
+import os
+
+# Load environment variables from .env file
+load_dotenv()
+
+# Retrieve the bot token and channel ID from the environment variables
+DISCORD_TOKEN = os.getenv('DISCORD_TOKEN')
+CHANNELID = int(os.getenv('CHANNELID'))
+# Define your API endpoint
+API_URL = 'http://127.0.0.1:5000/chatbot'
```

Comment:
Will you pls put the API_URL also in the env file.

Reply:
Yes doing it right away...